Sky Atlas/Sky Atlas.html

```diff
@@ -591,11 +891,13 @@ <h1>Articles</h1>
           <td>Distribution Of Agent Tokens</td>
           <td>Article</td>
           <td>
-            To the extent Sky invests capital in Agents and receives tokens in exchange, it will distribute those tokens to the following stakeholders:
+            To the extent Sky invests capital in Agents and receives tokens in exchange, it will distribute those tokens to the following
+            stakeholders:
             • USDS Token Rewards users
-            • SKY Activation users (see <dfn>A.0.1 - Atlas Preamble - Definitions - Activation</dfn>)
-            • SKY Seal users (see <dfn>A.0.1 - Atlas Preamble - Definitions - Sealing</dfn>)
-            The percentage of each token distributed to each of these stakeholders is determined by Sky Governance. In each case, to earn rewards users must opt to earn the token of a specific Agent, as opposed to either (1) the token of any other Agent, (2) the Sky Savings Rate in the case of USDS users, or (3) USDS Rewards in the case of SKY Activation or SKY Seal users.
+            • SKY Staking users (see <dfn>A.0.1 - Atlas Preamble - Definitions - Activation</dfn>)
```

Comment:
The definition title is changing to Staking

```suggestion
            • SKY Staking users (see <dfn>A.0.1 - Atlas Preamble - Definitions - Staking</dfn>)
```